[PLAY-326]: Configure automated package version upgrade

AntonNos opened the pull request.

dechegaray commented on package.json:
```diff
@@ -85,5 +87,10 @@
+  },
+  "husky": {
+    "hooks": {
+      "post-merge": "npm run version:minor"
```
Does this run on the CI? This will only run in your machine when manually merging, am I right?

AntonNos commented:
> I was thinking into this direction https://medium.com/@johnathanmiller/github-actions-npm-version-bump-using-labels-on-pr-merge-8c716a488a64 , basically bumping the version, only when a PR gets merged to master. This case, is using labels in the PR which is also smart

I removed old logic and added new `.yml` for version update
